deploy/olm-catalog/splunk/1.0.3/splunk.v1.0.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,350 @@
+          - list
+        - apiGroups:
+          - enterprise.splunk.com
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - statefulsets
+          verbs:
+          - list
+          - get
+          - watch
+        serviceAccountName: splunk:operator:resource-manager
+      deployments:
+      - name: splunk-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: splunk-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: splunk-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: splunk-operator
+                - name: RELATED_IMAGE_SPLUNK_ENTERPRISE
+                  value: docker.io/splunk/splunk:8.2.2
+                image: docker.io/splunk/splunk-operator:1.0.3
+                imagePullPolicy: IfNotPresent
+                name: splunk-operator
+                resources: {}
+              serviceAccountName: splunk-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - configmaps
+          - secrets
+          - pods
+          - pods/exec
+          - serviceaccounts
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - enterprise.splunk.com
+          resources:
+          - '*'
+          - indexerclusters
+          - clustermasters
+          - licensemasters
+          - searchheadclusters
+          - standalones
+          verbs:
+          - '*'
+        serviceAccountName: splunk-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
```

Comment:
Just a thought, I think at some point in future we should also support watching multiple namespaces(as in watching only specific namespaces rather than all namespaces)